### ISSUE TITLE
The history feeds wrongly remain displayed in the library after they are deleted using a time range until quit/resume

### ISSUE THREAD

**Issue by Softvision-RemusDranca** [MozillaReality/FirefoxReality#3042]
**[Affected Versions]:**
- Firefox Reality version 10-140842123 staging-master-signed.apk (28cd676).

**[Affected Platforms]:**
- HTC Vive Focus Plus - Android 7.1.2
- HTC Vive Plus - Android 7.1.2
- Pico Neo 2 - Android - 8.1.0
- Oculus Quest - Android 7.1.1
- Oculus Go - Android 7.1.2

**[Prerequisites]:**
- The headset is turned on and the app is opened.
- The History library has feeds logged.

**[Steps to reproduce]:**                                        
1. Click the History button from the tray.
2. Click the "Clear History" button and select the "Everything" radio button.
3. Click the "Clear Now" button and observe the behavior.

**[Expected results]:**
- All feeds are deleted and no longer displayed in the library.

**[Actual results]:**
- The feeds remain displayed until the app is quited and resumed.

**[Additional Notes]:**
- This issue is reproducible if any time range is selected from the Clear History prompt.
- This issue is intermittently reproducible with the Delete trash button as well.
- For the Pico and HTC devices the app needs to be force stopped and reopened for the feeds to no longer be displayed.
- Attached is a screen recording of the issue: [link](https://drive.google.com/open?id=1RCyXk8fo7n65Y7FjjIOgHG2GXxglwpCZ).

**Comment by keianhzo** [comment]
@Softvision-RemusDranca I can't reproduce this with the latest master, could you please verify?

**Comment by Softvision-RemusDranca** [comment]
I have retested this issue and it's no longer reproducible with the latest Firefox Reality version 10-141042250 (207f86b) build. The history is now deleted when deleted using a time range.
Tested on Oculus Go (Android 7.1.2), Oculus Quest (Android 7.1.1), HTC Vive Focus Plus (Android 7.1.2), HTC Vive Focus (Android 7.1.2), Pico Neo 2 (Android 8.1.0) and Pico G2 4K (Android 8.1.0).

Considering this I will close this issue as verified-fixed.
